code/modules/medical/genetics/bioEffects/hemochromia.dm

```diff
@@ -0,0 +1,278 @@
+ABSTRACT_TYPE(/datum/bioEffect/hemochromia)
+
+/datum/bioEffect/hemochromia_unknown
+	name = "Hemochromia Type-U"
+	desc = "A volatile mutation with 12 known stable alternatives. Will quickly break down into one of them if the subject exits the scanner."
+	id = "hemochromia_unknown"
+	probability = 111
+	effectType = EFFECT_TYPE_MUTANTRACE
+	msgGain = "You feel like you're out of breath."
+	msgLose = "You feel more stable."
+	reclaim_fail = 10
+	lockProb = 1
+	lockedGaps = 2
+	lockedDiff = 2
+	lockedChars = list("A","T","C","G")
+	lockedTries = 12
+	stability_loss = 5
+	icon_state  = "hemochromia_unknown"
+
+	OnLife()
```

Comment:
```suggestion
	OnLife()
		if(..()) return
```
This makes sure OnLife isn't ran if the mutation is removed or something. I'm not sure, whatever. Calling parents is a good idea either way.

Reply:
Done.